### ISSUE TITLE
Render time

### ISSUE THREAD

**Issue by creiser** [alievk/npbg#16]
Hello,

you mentioned that you can render a full HD image in 60 ms, which is impressive. Do you just render with PyTorch or do you use something like TensorRT for inference?

Thanks :)

**Comment by seva100** [comment]
Hi @creiser, no, this performance is simply recorded for our `viewer.py` (PyTorch only). You can check it by using the `--fps` option of the script (the performance will depend on the GPU used). Perhaps, with TensorRT it could be even faster!

**Comment by creiser** [comment]
Yes, definitely. We should give it a try and also use half precision. Real-time rendering really sets your technique apart from other recent methods
